js/base/Exchange.js

```diff
@@ -2585,7 +2585,14 @@ module.exports = class Exchange {
                 throw new BadSymbol (this.id + ' does not have market symbol ' + symbol);
             }
         }
-        throw new BadSymbol (this.id + ' does not have market symbol ' + this.json (symbol));
+        let symbolValue = symbol;
+        if (symbolValue === undefined) {
+            symbolValue = '';
+        }
+        else if (this.isObject (symbolValue) || Array.isArray(symbolValue)) {
+            symbolValue = this.json (symbolValue);
+        }
+        throw new BadSymbol (this.id + ' does not have market symbol ' + symbolValue);
```

Comment:
I think this is much heavier than we want, substituting it with an empty string is not the way we'd like to go.

Reply:
done